.github/workflows/pull-requests.yml

```diff
@@ -20,17 +20,8 @@ jobs:
             - name: Install dependencies
               run: yarn
 
-            - name: Compile contracts
-              run: yarn compile:contracts
-
-            - name: Build libraries
-              run: yarn build:libraries
-
-            - name: Run Prettier
-              run: yarn prettier
-
-            - name: Run Eslint
-              run: yarn lint
+            - name: Build | Lint | Format
```

Comment:
Should these changes be reflected in the prod workflow?

Reply:
completes successfully after re triggering it

Reply:
makes sense.
see https://github.com/privacy-scaling-explorations/zk-kit/pull/235/commits/25e11c1c01c050982e3ca33acb7ef17152dd7e10